tests/pkg/config/auditlog.go

```diff
@@ -1,9 +1,11 @@
 package config
 
 type AuditlogConfig struct {
-	TokenURL          string
-	ClientID          string
-	ClientSecret      string
-	ManagementURL     string
-	ManagementAPIPath string
+	TokenURL          string `envconfig:"AUDITLOG_TOKEN_URL"`
+	ClientID          string `envconfig:"AUDITLOG_CLIENT_ID"`
+	X509Cert          string `envconfig:"AUDITLOG_X509_CERT"`
+	X509Key           string `envconfig:"AUDITLOG_X509_KEY"`
+	ManagementURL     string `envconfig:"AUDITLOG_MANAGEMENT_URL"`
+	ManagementAPIPath string `envconfig:"AUDITLOG_MANAGEMENT_API_PATH"`
+	SkipSSLValidation bool   `envconfig:"AUDITLOG_SKIP_SSL_VALIDATION,default=false"`
```

Comment:
Correct me if I'm wrong but I think currently we execute auditlog tests on real env where maybe it's a good idea this property to be false and to not skip SSL validation?